pkg/apis/stork/v1alpha1/snapshotrestore.go

```diff
@@ -24,42 +23,40 @@ type VolumeSnapshotRestoreSpec struct {
 	DestinationPVC map[string]string `json:"pvcs,omitempty"`
 }
 
-// SnapshotRestoreStatusType is the status of volume in-place restore
-type SnapshotRestoreStatusType string
+// VolumeSnapshotRestoreStatusType is the status of volume in-place restore
+type VolumeSnapshotRestoreStatusType string
 
 const (
 	// VolumeSnapshotRestoreStatusInitial is the initial state when snapshot restore is initiated
-	VolumeSnapshotRestoreStatusInitial SnapshotRestoreStatusType = ""
+	VolumeSnapshotRestoreStatusInitial VolumeSnapshotRestoreStatusType = ""
 	// VolumeSnapshotRestoreStatusPending for when restore is in pending state
-	VolumeSnapshotRestoreStatusPending SnapshotRestoreStatusType = "Pending"
+	VolumeSnapshotRestoreStatusPending VolumeSnapshotRestoreStatusType = "Pending"
 	// VolumeSnapshotRestoreStatusRestore for when restore is in restore state
-	VolumeSnapshotRestoreStatusRestore SnapshotRestoreStatusType = "Restore"
+	VolumeSnapshotRestoreStatusRestore VolumeSnapshotRestoreStatusType = "Restore"
```

Comment:
This state is confusing. `Restore` implies it is in restore process. Maybe rename to `Restored`?

Reply:
restore stage is when snapshot restore objects are ready eg. downloaded